pt-BR/mail/chrome/messenger-newsblog/feed-subscriptions.dtd

```diff
@@ -1,5 +1,5 @@
 <!-- Subscription Dialog -->
-<!ENTITY newsBlogSubscriptions.label "RSS Subscriptions">
+<!ENTITY newsBlogSubscriptions.label "Assinaturas RSS">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY newsBlogSubscriptions.label "Subscrições RSS">